robottelo/virtwho_utils.py

```diff
@@ -316,6 +316,26 @@ def deploy_configure_by_script(
         return deploy_validation(hypervisor_type)
 
 
+def deploy_configure_by_command_check(command):
+    """Deploy and run virt-who servcie by the hammer command to check deploy log.
+
+    :param str command: get the command by UI/CLI/API, it should be like:
+        `hammer virt-who-config deploy --id 1 --organization-id 1`
+    :param str hypervisor_type: esx, libvirt, rhevm, xen, libvirt, kubevirt, ahv
+    :param str org: Organization Label
+    """
+    virtwho_cleanup()
+    try:
+        ret, stdout = runcmd(command)
```

Comment:
This is not a blocking comment, but I'd like to get you thinking in this direction @yanpliu.

Consider passing `target_sat` to the functions in this utils module, wherever you want to run `ssh.get_client().put`, or `runcmd.  Use the instance to call these functions directly on the targeted machine.

There are potential issues with the implicit nature of `runcmd` calls without a hostname, because `settings.server.hostname` isn't reliable within a test session as the intended target.